openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -77,6 +78,38 @@ class Config:
     endstate the torsion term is turned off/on depending on the state the
     unique atoms belong to. Default False.
     """
+    explicit_charge_correction = True
+    """
+    Whether to explicitly account for a charge difference during the
+    alchemical transformation by transforming a water to a counterion
+    of the opposite charge of the formal charge difference.
+
+    Please note that absolute charge changes greater than 1 are
+    currently not supported.
+    """
+    explicit_charge_correction_cutoff = 0.8 * unit.nanometer
+    """
+    The minimum distance from the system solutes from which an
+    alchemical water can be chosen. Default 0.8 * unit.nanometer.
+    """
+    alchemical_water_ion_positive_resname = 'NA'
+    """
+    Name of the positive ion used to replace the alchemical water with
+    if ``explicit_charge_correction`` is ``True`` and the absolute charge
+    difference is negative.
+    """
+    alchemical_water_ion_negative_resname = 'CL'
```

Comment:
Can these be pulled automatically from the SolventComponent solvation settings?

Reply:
done